key/aziot-keys/aziot-keys.h

```diff
@@ -33,7 +33,7 @@ typedef unsigned int AZIOT_KEYS_VERSION;
 /**
  * The base struct of all of function lists.
  */
-typedef struct {
+typedef struct AZIOT_KEYS_FUNCTION_LIST {
```

Comment:
These changes shouldn't be happening (in this PR). Make sure you're using the correct version of cbindgen. https://github.com/Azure/iot-identity-service/blob/9a12cd33ef27f82a5524c0d0d58938ec797fa852/ci/install-build-deps.sh#L157